packages/kubernetes/data_stream/container_logs/routing_rules.yml

```
@@ -0,0 +1,10 @@
+# Route container logs entries to the correct dataset and namespace based on the pod labels
+- source_dataset: kubernetes.container_logs
+  rules:
+    - target_dataset:
+        - "{{kubernetes.labels.elastic_co/dataset}}"
+        - "{{data_stream.dataset}}"
+      namespace:
+        - "{{kubernetes.labels.elastic_co/namespace}}"
+        - "{{data_stream.namespace}}"
+      if: "ctx?.kubernetes?.labels != null"
```

nit: `ctx` is always non-null
```suggestion
      if: "ctx.kubernetes?.labels != null"
```

---

Thanks! I knew this was unnecessary, but my hands kept typing it nonetheless for some reason! 🤦